### PR TITLE
NGDOC-210 Add upstream_origin_only

### PR DESCRIPTION
**Pull request by Jianghai-Quantil**
@nibinqtl @burtonquantil Added a new directive upstream_origin_only. Please help review.

### CODE REVIEW

File: en/docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -1235,6 +1235,16 @@ Indicates whether to look for each string to replace once or repeatedly. No chan
...
 
+### `upstream_origin_only`
+
+<span class="badge">standard</span> <span class="badge primary">Proprietary</span>
+
+**Syntax:** `upstream_origin_only on|off;` <br/>
+**Default:** `upstream_origin_only off` <br/>
+**Contexts:** server, location, if in location
+
+Enables or disables an origin only upstreaming. When set to on, a matching request is forwarded directly to the origin, bypassing any intermediate cache (including [origin shields](/cdn/apidocs#operation/get-cdn-shields)). This can be very useful if you would like some requests to go through intermediate layers while other requests (e.g. auth requests) to be forwarded directly to the origin.
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:1246]: I asked Ramprasad to clarify the functionality. Based on my understanding, I would like to modify to describe a bit more:

Enable or disable origin only upstreaming. When set to on, a matching request is forwarded directly to the origin, bypassing any intermediate cache including [origin shields](/cdn/apidocs#operation/get-cdn-shields).  For example, you may have configured your property's directConnection setting to "noDirect" to always use an intermediate cache or "auto" for us to dynamically determine the best course, by default.  The upstream_origin_only directive lets you bypass that setting and allow a subset of requests to go directly to the origin

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I suggest some additional detail to make it clearer to a customer the scenario when they may want to use this new directive.

File: en/docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (Jianghai-Quantil) [reply en/docs/edge-logic/supported-directives.md]: Suggestion accepted.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**